documentation/practices.md

```diff
@@ -62,6 +62,21 @@ For example, when a customer views a product, adds a product to the cart, or rem
 * The product and cart pages submit this information for further analysis by Google Analytics’ in a separate request.
 
 ### Global Product Attribute
-Google Analytics handlers use the global product attribute `Tag`. Use the formatted value of `store-name:category`, such as `vestri:Accessories` to submit the product category information. You can set the global attribute for each product in the catalog to avoid submitting empty values for the product's category to Google Analytics. 
+Google Analytics handlers use the global product attribute `Tag`. Use the formatted value of `store-name:category`, such as `vestri:Accessories` to submit the product category information. You can set the global attribute for each product in the catalog to avoid submitting empty values for the product's category to Google Analytics.
+
+## ARKit Quick Look Example
+
+**Apple ARKit Augmented Reality Quick Look Integration**
+
+The React PWA Reference Storefront is pre-configured for integration with Apple's ARKit web integration. As of iOS 12, you can incorporate 3D objects in to the real world using AR Quick Look directly through your Safari web browser when you visit a product's display page that supports the functionality. For more information on ARKit, see [Get ready for ARKit 2](https://developer.apple.com/arkit/).
```

Comment:
For iOS 12 or later versions
 directly through the Safari web browser 
For more information on ARKit, see [ARKit 2 documentation](https://developer.apple.com/arkit/).